### ISSUE TITLE
Update cookie banner to be modal

### ISSUE THREAD

**Issue by edthedev** [app-illinois/Design-Resources#168]
## Context

Being a modal is the expected accessible behavior, since users are expected to briefly interact with this information, the first time they visit a website.

This was blocked by #140, since we're not ready to do full cookie management, but #167 has a proposed work-around.

Blocked by #140, but may be activated after.

## Tasks

- [x] blocked by #152 
- [x] verify that our 'do not show for six months' cookie works on our test sites, after the new illinois based URL is in place (MVP 1.0.0 released)
- [x] update aria `role=dialog` to aria `role="alertdialog"`
- [x] un-comment the code we saved for allowing this to be modal
- [x] prevent scrolling while the banner is open
- [x] verify modal behavior
- [x] update the focus to start on the banner itself with tabindex = -1, rather than on the X button
- [x] prevent tab key navigation behind banner while banner is open

**Comment by edthedev** [comment]
Unblocked, as we have a six month cookie solution.

**Comment by edthedev** [comment]
Blocked by #152 . Need to load from an Illinois domain, to be able to set the cookie. May still be unable to set the cookie due to subdomain differences.

**Comment by edthedev** [comment]
This fix is deferred from  MVP 1.0.0 due to inability to test across the wide variety of sites where this change could cause usability, accessibility issues and disrupt work for all users (including screen reader users) if the fix for #140  doesn't work broadly. 

After we can verify that #140 works as intended, this planned change will be included in 1.1.0, to ensure compliance. 

Note: Lacking this update in the 1.0.0 release *does not reduce accessibility* compared to the current banner. The 1.0.0 release here is a substantial improvement over the current implemented vendor banner - per both accessibility tester feedback and our automated accessibility scans. This 1.0.0 update corrects several accessibility issues we have with the current vendor banner.

But - to reach our target of full compliance, and best serve our users - particularly those using screen readers -  this banner will become a modal, in the next release.

**Comment by edthedev** [comment]
The notice released earlier this week, and we have not had reports of the previous banner's zombie behavior - re-appearing on every page reload.

We do have to dismiss the banner more than we wish, as we must dismiss it once for each sub-domain. We are tracking that enhancement in #213.
